LUMO_toolbox/DOTHUB_LUMOdataQualityCheck.m

```diff
@@ -58,9 +64,91 @@ function DOTHUB_dataQualityCheck(nirsFileName,printFigFlag)
 dists = DOTHUB_getSDdists(SDtmp);
 
 % MOTION CHECK  #######################################################
-% - this script does not focus on motion content, but providing
+%timeSelectFlag allows the user to select a clean period (motion free) for analysis. 
+%A subset of channels are plotted to decrease computer load (this can
+%always be changed to plot all channels).
+if (timeSelectFlag==1) 
+    my_ans = 'n';
+    while my_ans == 'n';
```

Comment:
I don't understand why you need/want a While loop?  Is the idea that people can select a range until they get it right/are happy with it?

Reply:
Yeah. When I do this is isn't always clear where the cursor is selecting so having it highlight after helps me make sure that its not getting period with motion